compiler/typecheck/TcRnDriver.hs

```diff
@@ -2136,7 +2136,8 @@ getGhciStepIO = do
     let ghciM   = nlHsAppTy (nlHsTyVar ghciTy) (nlHsTyVar a_tv)
         ioM     = nlHsAppTy (nlHsTyVar ioTyConName) (nlHsTyVar a_tv)
 
-        step_ty = noLoc $ HsForAllTy { hst_bndrs = [noLoc $ UserTyVar (noLoc a_tv)]
+        step_ty = noLoc $ HsForAllTy { hst_bndrs = [noLoc $ -- TODO: I *think* Inferred is right here
```

Comment:
I think the choice is irrelevant, and I have no real idea what this code does. But I would favor `Specified`, for the reasons above.

Reply:
I tried to figure out what this code does, and I got totally stumped. Eventually we should probably find somebody who knows.